language-design.md

```diff
@@ -27,8 +41,17 @@ How would you know a well-designed language? What are the symptoms? How would
 you know a poorly designed language? What are the symptoms?
 
 **Response**
+I think one of the symptoms of a well designed language is that, as Guy Steele says, "new words defined by a library should look just like primitives of the language." In other words, the language should be seamless.
 
+I think a symptom of a poorly designed language is that there are many exceptions to the language rules. -Daniel
```

Comment:
Do you know any particular examples of a language with many exceptions to the language rules?